### PR TITLE
Replace Expand/Collapse wording with chevron svg

### PR DESCRIPTION
**Pull request by ras1**

### CODE REVIEW

File: src/components/PlaylistSelector.js [hunk 1]
```diff
@@ -53,4 +35,23 @@ function PlaylistSelector(props) {
   );
 }
 
+function onSelectNone(props) {
```
Comment (hebron-george) [inline src/components/PlaylistSelector.js:38]: Did these functions change or just moved down?

File: src/index.css [hunk 2]
```diff
@@ -63,6 +63,14 @@ body {background-color: #110f36;}
   height: 6em;
 }
 
+.chevron-down{
+  transform: rotate(0deg);
+}
+
+.chevron-down.flip{
+  transform: rotate(180deg);
```
Comment (hebron-george) [inline src/index.css:71]: 🔥🔥

Comment (hebron-george) [reply src/index.css:71]: Css God

**Review by hebron-george — APPROVED** [review]
LGTM but confused on why the history component was renamed

**Comment by hebron-george** [comment]
Can you add some screenshots to the PR description? I've been trying to do that for my PRs so that it's easier to see what changed when looking back (eg if a bug pops up in the future)

**Comment by ras1** [comment]
> LGTM but confused on why the history component was renamed

Renamed it because I kept not being able to find it, so I aligned naming with the PlaylistSelector

**Comment by ras1** [comment]
> Can you add some screenshots to the PR description? I've been trying to do that for my PRs so that it's easier to see what changed when looking back (eg if a bug pops up in the future)

This image shows the chevron open and closed

image

File: src/components/PlaylistSelector.js (hunk 1 above)
Comment (ras1) [reply src/components/PlaylistSelector.js:38]: just moved down

**Pull request merged**